### DISCUSSION TITLE
Install bcc/bpftrace failed with "Python pcp.pmda module is not installed, install it and try again"

### DISCUSSION THREAD

**Discussion by hsueh-yh** [bpftrace/bpftrace#3097] [Q&A] [answered]
Install [pmda](https://github.com/performancecopilot/pcp/tree/main/src/pmdas/bpftrace) bcc/bpftrace failed with "Python pcp.pmda module is not installed, install it and try again",
Does anyone know what specific modules need to be installed? Thanks!

```shell
[root@centos-8.3.2011-1]# pcp
Performance Co-Pilot configuration on centos-8.3.2011-1:

 platform: Linux 4.18.0-193.el8.x86_64 #1 SMP Wed Aug 4 03:09:24 PDT 2021 x86_64
 services: pmcd
     pmcd: Version 5.3.1-5, 10 agents, 1 client
     pmda: root pmcd proc pmproxy xfs linux mmv kvm jbd2 dm
 pmlogger: primary logger: /var/log/pcp/pmlogger/centos-8.3.2011-1/20240402.15.39
```

**Comment by danobi** [comment, marked as answer]
Hi, I recommend opening a ticket in the PCP repo. Most of the folks here are not very involved with PCP (so we don't know the answer to your question)
